src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/client/gui/occulus/OcculusAffinityTabRenderer.java

```diff
@@ -42,9 +53,11 @@ protected void renderFg(PoseStack pMatrixStack, int pMouseX, int pMouseY, float
         List<Component> drawString = new ArrayList<>();
         List<IAffinity> affinities = new ArrayList<>(registry.getValues());
         affinities.sort(null);
+        Player player = getPlayer();
+        assert player != null;
```

Comment:
Please do not use asserts, return instead.

Reply:
the assert is only there to shush the possible NPE warning it would not throw in prod anyway